ThunderCloud/LocalisationsTableViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LocalisationsTableViewController.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 07/09/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import UIKit
+import ThunderTable
+
+class LocalisationsTableViewController: TableViewController {
+    
+    override func viewDidLoad() {
+        
```

Comment:
We need to have a word about you adding a space from the function to the `super` call! 🤣

Reply:
I have a habit, and not sure where it's come from exactly... of always including a white-space line if a method is over a certain size :P See the whole of this file... yet in `LocalisationsTool.swift` I've not put a white-space line in `launchUI(in:)`! Is there anything in the swift style guidelines about it?

Reply:
Haha nothing I'm aware of, but maybe! 😛